src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+const Navbar = () => {
+  return (
+    <nav className="nav nav-boxshadow">
+        <a href="/" className="brand my-brand">
+            <div className="logo">
+                <img loading="lazy" src="./images/nav/nav-logo.png" alt="logo" />
+            </div>
```

Comment:
suggestion - don't use loading="lazy" in navbar as it is the first thing which will be displayed to the user. You can use it in places which the user will not see immediately.